src/packages/user/user/workspace/components/user-workspace-info/user-workspace-info.element.ts

```diff
@@ -0,0 +1,138 @@
+import { getDisplayStateFromUserStatus } from '../../../../utils.js';
+import { UMB_USER_WORKSPACE_CONTEXT } from '../../user-workspace.context.js';
+import { html, customElement, state, css, repeat, ifDefined } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UserResponseModel } from '@umbraco-cms/backoffice/backend-api';
+
+type UmbUserWorkspaceInfoItem = { labelKey: string; value: string | number | undefined };
+
+@customElement('umb-user-workspace-info')
+export class UmbUserWorkspaceInfoElement extends UmbLitElement {
+	@state()
+	private _user?: UserResponseModel;
+
+	@state()
+	private _userInfo: Array<UmbUserWorkspaceInfoItem> = [];
+
+	#userWorkspaceContext?: typeof UMB_USER_WORKSPACE_CONTEXT.TYPE;
+
+	constructor() {
+		super();
+
+		this.consumeContext(UMB_USER_WORKSPACE_CONTEXT, (instance) => {
+			this.#userWorkspaceContext = instance;
+			this.observe(this.#userWorkspaceContext.data, (user) => {
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.